Users/abehenderson/textfiles/log.txt

```diff
@@ -0,0 +1,4 @@
+I used mkdir Users/abehenderson/textfiles to create a new directory 'textfiles'.
+I used touch Users/abehenderson/textfiles/log.txt to create this file 'log.txt'
+I could not find a terminal command to open log.txt in the editor window, so I clicked on it with my mouse.  
+Did I cheat?
```

Comment:
Yes! Look for a new issue ...

Reply:
Ok, so is issue #27 Opening files in Terminal linked to this issue now?

Reply:
or is it separate?

Reply:
Not linked, just a new issue dealing specifically with what you brought up on line 3.